buildconfig/pygame-stubs/pixelcopy.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Optional
+import numpy
+
+from pygame.surface import Surface
+
+def surface_to_array(array: numpy.ndarray, surface: Surface, kind: Optional[str]='P', opaque: Optional[int]=255, clear: Optional[int]=0) -> None: ...
```

Comment:
`array` here seems to be anything that `pgObject_GetBuffer` works on, not just `numpy.ndarray`.

Reply:
`kind` can be one of these different characters.

From the docs:
> Recognized kind values are the single character codes 'P', 'R', 'G', 'B', 'A', and 'C'. Kind codes are case insensitive, so 'p' is equivalent to 'P'.

Reply:
For now I can replace it with Any, to get if something works on something I would have to use typing.Protocol, maybe in future I will add it, Kind is fixed